src/pages/linkme.php.js

```diff
@@ -75,7 +75,8 @@ function LinkmePhp() {
 
   // do we scroll to edition ? - this one is for loans on profile page where we do NOT want
   // to scroll to specific edition - see @components/profile/utils.js::getWorkUrlForProfile
-  const scrollToEdition = router?.query?.["scrollToEdition"] || false;
+  const scrollToEdition =
+    router?.query?.["scrollToEdition"] === "true" || false;
```

Comment:
du kan skrive `router?.query?.scrollToEdition` i stedet for at give den som en string